polling-app/controller/polls.js

```diff
@@ -26,23 +26,42 @@ router.route('/')
   .post(function(req,res) {
       // Code to add new polls.
       console.log("/post called")
-      var pollObject = new pollModel();
+
       // Calling our model function.
       // make sure to set content-type header to content-type:application/json
       var data = {
         question: req.body.question,
         options: req.body.polls
       }
-      console.log(typeof req.body.polls)
-      console.log(data) 
-      pollObject.addNewPolls(data, function (err){
-           if(err) {
-                response = {"error" : true,"message" : "Error adding poll data"};
-           } else {
-                response = {"error" : false,"message" : "Poll added"};
-           }
-           res.json(response);
-      });
+      
+      //validate 
+      if(!(data.question === undefined ) || !(data.options === undefined )){

```

Comment:
Can we improve this if condition to handle the validation stage? to use a single if condition ending with a return rather than if..else.